RUFAS/biophysical/animal/nutrients/beef_cow_calf_requirements_calculator.py

```
@@ -0,0 +1,472 @@
+            Nutrition requirements including pregnancy, lactation, and growth terms.
+
+        Raises
+        ------
+        ValueError
+            If inputs.animal_type is not a beef cow-calf type.
+
+        """
+        if inputs.animal_type not in _BEEF_COW_CALF_TYPES:
+            raise ValueError(
+                f"BeefCowCalfRequirementsCalculator only handles cow-calf types; " f"got {inputs.animal_type.value}."
+            )
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)